patches/chrome-browser-profiles-profile_info_cache.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/profiles/profile_info_cache.cc b/chrome/browser/profiles/profile_info_cache.cc
+index c2decc6db935a7052c80cfb493ae2cc146dbef78..dc3d9b764d8e45d2fe9a4c932ee3d3bbf4640f0c 100644
+--- a/chrome/browser/profiles/profile_info_cache.cc
++++ b/chrome/browser/profiles/profile_info_cache.cc
+@@ -63,6 +63,7 @@ void DeleteBitmap(const base::FilePath& image_path) {
+ ProfileInfoCache::ProfileInfoCache(PrefService* prefs,
+                                    const base::FilePath& user_data_dir)
+     : ProfileAttributesStorage(prefs), user_data_dir_(user_data_dir) {
++  BRAVE_DISABLE_AVATAR_DOWNLOAD;
```

Comment:
we don't need to patch for this, just call `GetProfileInfoCache().
      set_disable_avatar_download_for_testing(true)` in `BraveProfileManager` initializer

Reply:
or better yet do a chromium_src override of ProfileAvatarDownloader -> BraveProfileAvatarDownloader that just immediately calls the callback on `Start` and passes an empty gfx::Image